apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -484,6 +484,13 @@
         "CANCEL": "Cancel",
         "SUCCESS_DELETE": "Attributes deleted successfully",
         "SUCCESS_SAVE": "Attributes saved successfully"
+      },
+      "RESOURCE_ATTRIBUTES": {
+        "TITLE": "Attributes",
+        "DELETE": "Remove selected",
+        "FILTER": "Filter by id, name, value or description",
```

Comment:
I don't think it is actually filtering by id. At least for me, it doesn't work. Also, I would start the label with `Filter resources by...`